spec/lib/hubspot/task_spec.rb

```diff
@@ -0,0 +1,44 @@
+RSpec.describe Hubspot::Task do
+  describe 'create!' do
+    subject(:new_task) do
+      params = {
+        hs_task_body: 'i am a task',
+        hs_task_subject: 'title of task',
+        hs_timestamp: DateTime.now.strftime('%Q')
+      }
+      described_class.create!(params, ticket_id: 16174569112)
+    end
+
+    it 'creates a new task with valid properties' do
+      VCR.use_cassette 'task' do
+        expect(new_task.id).not_to be_nil
+        expect(new_task.properties[:hs_task_status]).to eq('NOT_STARTED')
+        expect(new_task.properties[:hs_task_subject]).to eq('title of task')
+        expect(new_task.properties[:hs_body_preview]).to eq('i am a task')
+      end
+    end
+  end
+  
+  describe 'find' do
+    let(:task_id) { 64075014222 }
+
+    subject(:existing_task) { described_class.find(task_id, 'hs_task_subject,hs_task_status') }
```

Comment:
is there a way to make the second parameter explicit in some way? I don't understand it now :sweat_smile:

Reply:
according to doc it's list of comma separated properties name we want returned. But I don't see any doc related comments on the gem so I don't know if I should put a comment. Renaming paramter to `string_property_list` could also be done if you prefer

Reply:
Should there be a default value with the default fields?

Reply:
also should we support an array that will get `join`ed ?